internal/csi-addons/rbd/volumegroup.go

```diff
@@ -22,16 +22,17 @@ import (
 	"fmt"
 	"slices"
 
-	"github.com/ceph/ceph-csi/internal/rbd"
-	"github.com/ceph/ceph-csi/internal/rbd/group"
-	"github.com/ceph/ceph-csi/internal/rbd/types"
-	"github.com/ceph/ceph-csi/internal/util/log"
-
+	librbd "github.com/ceph/go-ceph/rbd"
 	"github.com/csi-addons/spec/lib/go/replication"
 	"github.com/csi-addons/spec/lib/go/volumegroup"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+
+	"github.com/ceph/ceph-csi/internal/rbd"
+	"github.com/ceph/ceph-csi/internal/rbd/group"
+	"github.com/ceph/ceph-csi/internal/rbd/types"
+	"github.com/ceph/ceph-csi/internal/util/log"
```

Comment:
ceph-csi prefers local imports before 3rd party ones (for whatever unknown to me reason). This is an unrelated change that should be dropped.